src/main/java/springsamurais/toyswapbackend/model/Comment.java

```diff
@@ -1,3 +1,78 @@
 package springsamurais.toyswapbackend.model;
 
-public class Comment {}
+import jakarta.persistence.*;
+import java.util.Date;
+
+@Entity
+//@Table(name = "comments")
```

Comment:
HI, can you keep the table name matching the class name for consistency purposes?
Change it to _COMMENT_

Reply:
Hi , Thank you for the review. changes made and push. please let me know for any changes. Thank you